awscrt/mqtt5.py

```diff
@@ -41,6 +41,13 @@ class QoS(IntEnum):
     Note that this client does not currently support QoS 2 as of (August 2022)
     """
 
+    def to_mqtt3(self):
+        from awscrt.mqtt import QoS as Mqtt3QoS
+        """Convert a Mqtt5 QoS to Mqtt3
+
+        """
+        return Mqtt3QoS(self._value_)
```

Comment:
Trivial: `_value_` member is kind of a private member and can be used for setting a new value. `self.value` would be more appropriate here, I think